data/mods/Aftershock/maps/mapgen/houses/houses.json

```diff
@@ -0,0 +1,76 @@
+[
+  {
+    "method": "json",
+    "om_terrain": "afs_shelter_1",
+    "type": "mapgen",
+    "weight": 100,
+    "object": {
+      "fill_ter": "t_metal_floor",
+      "rows": [
+        "                        ",
+        "                        ",
+        "   ++o+++o++o+++ppp.ppp ",
+        "   +@.x.x.@+.@++......p ",
+        "   +D.+.+.D+.D+++++l+++ ",
+        "   ++n+.++n+xn++q+...o  ",
+        "   +D.x......hf+x++l+++ ",
+        "   o@.+......hfn.77.77+ ",
+        "   +n++++x++++n+......o ",
+        "   o...........x......+ ",
+        "   +nx++nx++nx++...++++ ",
+        "   +5.t+5.t+5.t+...+    ",
+        "   ++9+++9+++9++...o    ",
+        "   ++++++++++++....+    ",
+        "   +{L.............++++ ",
+        "   oF..........H.....y+ ",
+        "   oV.J........H..2..y+ ",
+        "   oI.J........H..2..y+ ",
+        "   +P.S........H.....y+ ",
+        "   +M^M...E.......Y...+ ",
+        "   +++++R.......+++++++ ",
+        "       +RRRR.JWZ+       ",
+        "       ++++++ooo+       ",
+        "                        "
+      ],
+      "palettes": [ "standard_afsshelter_palette" ],
+      "place_item": [ { "item": "television", "repeat": 1, "x": 21, "y": 17 }, { "item": "toaster", "repeat": 1, "x": 6, "y": 17 } ],
```

Comment:
Is `repeat: #` always needed? If it's a just-in-case, that's fine, of course.

Reply:
Just mostly copy and pasted from the regular house palette, no idea if it is really?

Reply:
You dont actually need repeat in this case.